.yamato/urp_performance_boatattack-android-opengles3.yml

```diff
@@ -84,7 +84,7 @@ Build_URP_Performance_BoatAttack_Android_OpenGLES3_performance_playmode_runtime_
       -  cd TestProjects/BoatAttack && unity-config project add dependency "com.unity.shadergraph@file:../../../com.unity.shadergraph" --project-path .
       -  cd TestProjects/BoatAttack && unity-config project add dependency com.unity.addressables@1.16.7 --project-path .
       -  cd TestProjects/BoatAttack && unity-config project add dependency com.unity.scriptablebuildpipeline@1.11.2 --project-path .
-      -  cd TestProjects/BoatAttack && unity-config project add dependency com.unity.test-framework@1.1.18 --project-path .
+      -  cd TestProjects/BoatAttack && unity-config project add dependency com.unity.test-framework@1.1.29 --project-path .
```

Comment:
Thanks for this!  🤦🏻‍♂️  I think I forgot to regenerated the yml files in my previous PR https://github.com/Unity-Technologies/Graphics/pull/5741.....

Reply:
Causing the URP Perf tests to continue to fail

Reply:
Oh, I didn't notice it myself :sweat_smile: but I'm glad!